src/lib.rs

```diff
@@ -1415,6 +1415,13 @@ where
     }
 }
 
+///
+/// Server Error Response
+///
+/// See `StandardErrorResponse` for an implementation adhering to the oauth standard.
```

Comment:
nit: OAuth

I think it's worth mentioning here that the trait exists to support interoperability with non-compliant providers. something similar to the rustdoc for `TokenResponse`.